README.md

```diff
@@ -10,7 +10,14 @@
 
 First, install all [build dependencies](docs/dev/dependencies.md).
 
-Clone this repository to `src/github.com/openshift/installer` in your [GOPATH](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable). Then build the `openshift-install` binary with:
+Clone this repository to `src/github.com/openshift/installer` in your [GOPATH](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable). 
+
+```sh
+mkdir -p ${GOPATH}/src/github.com/openshift/installer
+git clone https://github.com/openshift/installer ${GOPATH}/src/github.com/openshift/installer
```

Comment:
These assume your `GOPATH` has a single path, and that it's set at all.  See also previous discussion starting [here][1].  Currently if you clone *anywhere* and run `hack/build.sh`, it should either work (clones into your `GOPATH`) or suggest where you should move the checkout to (clones out of your `GOPATH`).  I don't know if it's worth giving pre-`build.sh` advice in English duplicating that logic.

[1]: https://github.com/openshift/installer/pull/528#discussion_r228454665

Reply:
Cloning _anywhere_ didn't work for me? I had to clone into the $GOPATH -- which for me doesn't point to many paths..

Reply:
> Cloning anywhere didn't work for me?

What did `build.sh` say when it failed?

Reply:
@wking I suppose the question is, should the README.md capture every possible nuance, or should it just be an example of how to kickstart the project. 

As someone new to Go/OpenShift Installer I had no clue the reference the GOPATH, so I figured I would add to the README.md what I did to make things work for me, as someone new to both Go and OpenShift.